packages/ui/src/forum/components/category/storybook-helpers.ts

```diff
@@ -0,0 +1,37 @@
+import { sub } from 'date-fns'
+import faker from 'faker'
+
+import { asBlock } from '@/common/types'
+import { ForumModerator, ForumPost, ForumThread } from '@/forum/types'
+import { getMember } from '@/mocks/helpers'
+
+export const asModerator = (hasHandle = true, hasAvatar = false) => {
+  return (index: number): ForumModerator => ({
+    id: String(index),
+    membershipId: '0',
+    handle: hasHandle ? faker.name.firstName() : undefined,
+    avatar: hasAvatar ? faker.image.avatar() : undefined,
+  })
+}
+
+export const asPost = (text: string): ForumPost | undefined => {
+  if (text)
+    return {
+      id: '0',
+      createdAt: sub(Date.now(), { minutes: 25 }).toISOString(),
+      createdAtBlock: asBlock(),
```

Comment:
There should be a helper now for this. I've created something like `generateRandomBlock()` so you can find this and re-create for storybook purposes.